src/cider/nrepl/middleware/inspect.clj

```diff
@@ -61,48 +69,39 @@
   [handler msg]
   (handler (eval-msg msg)))
 
-(defn- success [{:keys [transport] :as msg} inspector]
-  (transport/send transport (response-for msg :value (:rendered inspector) :status :done)))
-
-(defn- failure [{:keys [transport] :as msg} err err-kw]
-  (transport/send transport (base-error-response msg err err-kw :done)))
+(defn- success [msg inspector]
```

Comment:
I guess we can give this some more meaningful name now - e.g. `inspector-response`.